pages/common/borg.md

```diff
@@ -0,0 +1,32 @@
+# borg
+
+> Deduplicating backup tool with compression and encryption.
+> Supports local and remote backups, mountable as filesystems.
+
+- Initialise a (local) repository:
+
+`borg init {{/path/to/repo/folder}}`
+
+- Backup a folder into the repository, creating an archive called Monday:
+
+`borg create --progress {{/path/to/repo/folder::Monday}} {{/path/to/source/folder}}`
```

Comment:
Are the `::` required here? Perhaps we could tweak this to `{{/path/to/repo/folder}}::{{Monday}}` if that's the case?

Reply:
Yeah, the :: are part of the syntax. I like the suggestion of separating the archive name.